securedrop/tests/functional/test_make_account_changes.py

```diff
@@ -1,24 +0,0 @@
-# -*- coding: utf-8 -*-
-from . import functional_test, journalist_navigation_steps
-
-
-class TestMakeAccountChanges(
-    functional_test.FunctionalTest, journalist_navigation_steps.JournalistNavigationStepsMixin
-):
-    def test_admin_edit_account_html_template_rendering(self):
```

Comment:
This wasn't really testing anything specific, and was duplicating other existing tests.